msgraphforentra.json

```
@@ -0,0 +1,1008 @@
+{
+    "appid": "bf5de8fd-6437-4fc9-8c33-5f5bd3971d26",
+    "name": "MS Graph for Entra",
+    "description": "This app integrates with Microsoft Entra to execute various generic and investigative actions",
+    "type": "endpoint",
+    "product_vendor": "Microsoft",
+    "logo": "logo_msgraphforentra.svg",
+    "logo_dark": "logo_msgraphforentra_dark.svg",
+    "product_name": "Microsoft Entra",
+    "python_version": "3",
+    "product_version_regex": ".*",
+    "publisher": "Splunk",
+    "contributors": [
+        {
+            "name": "Ian Wills"
+        }
+    ],
+    "license": "Copyright (c) 2022-2024 Splunk Inc.",
+    "app_version": "1.0.0",
+    "utctime_updated": "2024-12-17T19:23:33.000000Z",
+    "package_name": "phantom_msgraphforentra",
+    "main_module": "msgraphforentra_connector.py",
+    "min_phantom_version": "6.1.1",
```

Technically, our static tests enforce that the `min_phantom_version` be no greater than "latest - 1", so we should default to that.
```suggestion
    "min_phantom_version": "6.3.0",
```

---

However, if we have customers on SOAR 6.1.1 requesting this connector, I don't mind ignoring that

---

I'm okay with either way on this. Customer isn't on 6.3.0+ yet, but we can always tell the installer to ignore the min version check with:

<PHANTOM_HOME>/bin/phenv python -m manage toggle_min_phantom_version_check --disable